docs/guides/webview-versions.md

```diff
@@ -0,0 +1,190 @@
+# Webview Versions
+
+## WebView2 (Edge/Chromium)
+
+Tauri uses WebView2 which is based on Microsoft Edge and therefore Chromium. WebView2 can update itself, you are guaranteed a relatively recent chromium build on all Windows targets.
+
+WebView2 is supported on Windows 7 and newer and comes preinstalled on Windows 11. On versions older than Windows 11 the installer generated by Tauri takes care of ensuring WebView2 is installed on the system.
+
+## WebKit
+
+Tauri uses WebKit on macOS (through [WKWebView](https://developer.apple.com/documentation/webkit/wkwebview?language=objc)) and on Linux (through webkit2gtk).
+
+Webkit version numbers are quite complicated, so below is some helpful information to understand them.
+
+<details>
+<summary>How to read WebKit version numbers</summary>
+
+WebKit version numbers are made up of 5 segments and a numeric prefix indicating the OS WebKit is built for. The prefix is called the `SYSTEM_VERSION_PREFIX` and seems to be only present for macOS and iOS builds, not for Linux. Furthermore, if the last two segments are both `0` they can be omitted; So a version like `613.2.7.0.0` would be referred to as `613.2.7`.
+
+`$(SYSTEM_VERSION_PREFIX)$(MAJOR_VERSION).$(MINOR_VERSION).$(TINY_VERSION).$(MICRO_VERSION).$(NANO_VERSION)`
+
+### `SYSTEM_VERSION_PREFIX`
+
+Here is what the `SYSTEM_VERSION_PREFIX` values mean:
+
+| macOS version | SYSTEM_VERSION_PREFIX |
+| ------------- | --------------------- |
+| 10.11         | 11                    |
+| 10.12         | 12                    |
+| 10.13         | 13                    |
+| 10.14         | 14                    |
+| 10.15         | 15                    |
+| 11.0          | 16                    |
+| 12.0          | 17                    |
+| 13.0          | 18                    |
+| sdk=iphone\*  | 8                     |
```

Comment:
Is the "*" for anything? I don't see something later in the page for it

Reply:
That's taken straight from the WebKit Source 🤷‍♂️ I guess we can also remove that row entirely until we have proper iOS support